torch/csrc/jit/passes/lower_tuples.cpp

```diff
@@ -17,6 +19,12 @@ std::unordered_set<Symbol> white_list = {
   prim::Return,
 };
 
+int64_t indexVal(Value * index) {
+  return (*toIValue(index->node()->output())).toInt();
```

Comment:
This is unsafe if `index` is not a constant. It would be better to use`.value().toInt()` which would throw in that case. The check doesn't cost much but will save us a lot of time in the future if this invariant were to change (the current code will read uninitialized memory silently).